pkg/engine/armparams.go

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package engine
+
+//
+//import (
```

Comment:
what's going on with all this commented code?

Reply:
As we had discussed earlier, I had eventually chosen to stick to kubernetesparams.t instead of writing the entire go code equivalent of that. It was an interim decision though.

If we decide on going forward with parameters in Go, I can uncomment this code and continue working on top of this.

Reply:
Hmmm, can you convert this to an issue or a markdown file in the design doc directory? Let's not actually make this a part of the diff (and I'd advocate we don't want this in the code as well, at least not in this directory)